integrations/Posthog/browser.js

```diff
@@ -16,6 +16,7 @@ class Posthog {
     this.propertyBlackList = [];
     this.xhrHeaders = {};
     this.enableLocalStoragePersistence = config.enableLocalStoragePersistence;
+    this.loaded = config.loaded; // Function that will be run once PostHog loads
```

Comment:
The ideal way will be to create a callback function within our JS SDK load function. For example `rudderanalytics.load(writeKey, dataplaneURL, {
       integrations: {
         All: true,
         POSTHOG: {
          // callback function
         },
       },
 });`

and this should be accessible from the already implemented file.